model/monitorCache/implementation/MonitorCacheService.php

```diff
@@ -90,10 +90,6 @@ protected function prepareCondition($condition, &$parameters, &$selectClause)
     {
         $whereClause = '';
 
-        if (is_array($condition) && count($condition) === 1 && is_array(current($condition))) {
```

Comment:
This condition is necessary there. In case if a criteria is something like that:
```php
$deliveryMonitoringService->find([
    ['status' => 'finished']
]);
```
Without this condition Unit tests are fails.